### ISSUE TITLE
Soap unidentified index

### ISSUE THREAD

**Issue by sunnycs121** [br0xen/ConnectWise_PHP#5]
Hello

Is this library made for soap calls? if yes is there any library that can be used for REST API calls for ConnectWise? as i am sending rest get request and i am getting an error saying 

"Undefined index: soap" here:
`$retObj = $xmlObj->children($name_spaces['soap'])->Body->children();`

Please help

**Comment by br0xen** [comment]
Yup, this project was built a couple of years ago for a project I was
working on when they only supported SOAP. I haven't had to work with
Connectwise since then. I have no idea if anyone else or there has built a
restful library for it.

Sorry, and good luck!
